docs/available_commands.md

```diff
@@ -165,8 +165,8 @@ Where name, version, hash and timestamp are information of the `metadata` from a
 
 If it is desired to have a different Regex, this can be done via the flag `--filename-pattern`, i.e.:
 
-    cdx-ev validate mybom.json --filename-pattern=".*" # every character allowed
-    cdx-ev validate mybom.json --filename-pattern="(^bom\.json$)" # only bom.json allowed
+    cdx-ev validate bom.json --filename-pattern=".*" # every character allowed
```

Comment:
I would prefer to remain the "mybom.json" to show that this way also a different name for the bom could be used.

Reply:
ok, changed that

Reply:
I was just referring to L#168, I was fine with L#169 being changed to "bom.json", so that a valid file name would be provided for the validation 😁